docs/specs/sequences/endpoint_reconfiguration.md

```diff
@@ -0,0 +1,150 @@
+[**UDMI**](../../../) / [**Docs**](../../) / [**Specs**](../) / [**Sequences**](./) / [Endpoint Reconfiguration](#)
+
+# Endpoint Reconfiguration
+
+Endpoint reconfiguration is the reconfiguration of the UDMI endpoint through the
+UDMI blob delivery mechanisms via UDMI config messages.
+
+The [endpoint configuration blob](https://github.com/faucetsdn/udmi/blob/master/tests/configuration_endpoint.tests/simple.json) is a JSON object defined by
+[configuration_endpoint.json](https://faucetsdn.github.io/udmi/gencode/docs/configuration_endpoint.html), which is base64 encoded in the config message.
+
+
+## Tests
+
+### Valid Endpoint (Successful Reconfiguration)
+
+**Notes**
+- `<ENDPOINT>` is a **base64** encoded endpoint object
+- `blobset.blobs_iot_endpoint_config` block is only present in a device's state messages when it is present in the last received config message
+
+```mermaid
+%%{wrap}%%
+sequenceDiagram
+    autonumber
+    participant D as Device
+    participant E as Original Endpoint
+    participant E' as New Endpoint
+    E->>D:CONFIG MESSAGE<br>timestamp = t0<br/>blobset.blobs._iot_endpoint_config = null (unset)
+    D->>E:STATE MESSAGE<br/>system.last_update = t0<br/>blobset.blobs._iot_endpoint_config = null (unset)
+    E->>D:CONFIG MESSAGE<br>timestamp = t1<br/>blobset.blobs._iot_endpoint_config.base64 = <ENDPOINT><br>blobset.blobs._iot_endpoint_config.phase = "final"
+    D->>E:STATE MESSAGE<br/>system.last_update = t1<br/>blobset.blobs._iot_endpoint_config.phase = "apply"
+    D-->>E':CONNECTION ATTEMPT
```

Comment:
I'd put this in a "<30s" loop box as well, for consistency with the other diagram.  It doesn't need anything else inside it, but it clarifies the overall structure, I think.

Reply:
Put into a loop box